src/recommendpolicy/recommendPolicy.go

```diff
@@ -52,6 +52,11 @@ type LabelMap = map[string]string
 // DeployNsName stores the identified deployments in a namespace
 var DeployNsName []types.Deployment
 
+// NsNotFilter stores the namespace which needs to be filtered out
+var NsNotFilter = cfg.CurrentCfg.ConfigSysPolicy.NsNotFilter
+
+var policies []types.KnoxSystemPolicy
```

Comment:
This is a very bad practice to keep the variables global. The functions using `policies` can no more be used in multi-threaded envs.

`policies` needs to be local vars.

Reply:
handled the bad func, PTAL